int32 nameid
int32 x
int32 y
int32 z
int32 width
int32 height
